skills/CEX/okx.py

```diff
@@ -0,0 +1,70 @@
+rom .base import CEXBaseSkill
+
+class TradeOnOkx(CEXBaseSkill):
```

Comment:
_run and _arun is required for a skill.

args_schema is required too, if there is no param, please use an empty pydantic object, for example see twitter/mentions.py